# Solaris-specific tweaks to the rbash(1) man page.
# Also, say the correct bash version and release date.
# Not for upstream.
--- doc/rbash.1	2008-08-12 06:24:51.000000000 -0700
+++ doc/rbash.1	2010-09-01 10:52:04.397206373 -0700
@@ -3,8 +3,8 @@
 rbash \- restricted bash, see \fBbash\fR(1)
 .SH RESTRICTED SHELL
 .nr zY 1
-.so bash.1
+.so /usr/share/man/man1/bash.1
 .SH SEE ALSO
 bash(1)
